pallets/settlement/src/lib.rs

```diff
@@ -22,8 +22,8 @@
 //!
 //! - `create_venue` - Registers a new venue.
 //! - `add_instruction` - Adds a new instruction.
-//! - `authorize_instruction` - Authorizes an existing instruction.
-//! - `unauthorize_instruction` - Unauthorizes an existing instruction.
+//! - `affirm_instruction` - Provides affirmation to an existing instruction.
+//! - `withdraw_instruction` - Deny an existing instruction.
```

Comment:
`withdraw_instruction` is not correct. The function withdraws the `affirmation`, not the instruction.